controllers/sriovoperatorconfig_controller.go

```diff
@@ -109,6 +109,44 @@ func (r *SriovOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.
 		return reconcile.Result{}, err
 	}
 
+	// Fetch the SriovNetworkNodePolicyList
+	policyList := &sriovnetworkv1.SriovNetworkNodePolicyList{}
+	err = r.List(ctx, policyList, &client.ListOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			// Request object not found, could have been deleted after reconcile request.
+			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
+			// Return and don't requeue
+			return reconcile.Result{}, nil
+		}
+		// Error reading the object - requeue the request.
+		return reconcile.Result{}, err
+	}
+
+	defaultPolicy := &sriovnetworkv1.SriovNetworkNodePolicy{}
+	err = r.Get(ctx, types.NamespacedName{Name: consts.DefaultPolicyName, Namespace: namespace}, defaultPolicy)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			// Default policy object not found, create it.
```

Comment:
do we want to create the policy here ? shouldnt the node policy controller create it ?

Reply:
fixed